paper/basic_training.tex

```diff
@@ -616,28 +620,52 @@ \subsubsection{Popular Thermostats}
 
 \subparagraph{Deterministic Thermostats}
 \begin{enumerate}[listparindent=\parindent]
-    \item \textbf{Velocity Rescaling}
+    \item \textbf{Simple Velocity Rescaling}
 
-        The velocity rescaling thermostat is one of the simplest thermostats to implement, however, this thermostat is also one of the most non-physical thermostats as well.
+        The simple velocity rescaling thermostat is one of the easiest thermostats to implement, however, this thermostat is also one of the most non-physical thermostats as well.
 
          This thermostat relies on rescaling the momenta of the particles every $N$ timesteps based on their ratio of the instantaneous temperature to the target temperature\cite{thermostatAlgorithms2005}. 
          This leads to multiple issues when trying to sample data using this thermostat.
          First, this method is not reversible, there lacks a way for the particles to have knowledge of their previous thermal history.
          This makes any dynamical value impossible to measure (diffusion for example). 
          There is also no distribution of momenta; this thermostat does not sample the canonical distribution and cannot be used for sampling at thermal equilibrium.
+         The isokinetic (constant kinetic energy) ensemble samples the
```

Comment:
A "one sentence per line" layout works really well with git diff logs as it makes it easier to see changes/edits, if you're willing to adjust your layout (or your editor) to accommodate that. Specifically, it makes revisions a lot easier to check in subsequent PRs.

Reply:
Sure. Fixing it soon.